cea/datamanagement/format_helper/format_helper.py

```
@@ -18,22 +20,28 @@
 __email__ = "cea@arch.ethz.ch"
 __status__ = "Production"
 
+from cea.datamanagement.format_helper.cea4_verify_db import cea4_verify_db, path_to_db_file_4
```

Ruff (0.8.2)

`cea.datamanagement.format_helper.cea4_verify_db.path_to_db_file_4` imported but unused

Remove unused import: `cea.datamanagement.format_helper.cea4_verify_db.path_to_db_file_4`

(F401)